modules/nextflow/src/main/groovy/nextflow/k8s/K8sTaskHandler.groovy

```diff
@@ -16,6 +16,9 @@
 
 package nextflow.k8s
 
+import nextflow.exception.K8sOutOfCpuException
+import nextflow.exception.K8sOutOfMemoryException
```

Comment:
out of disk space too?

Reply:
we used shared storage so running out of disk space shouldn't be an issue